frontends/web/src/routes/account/info/info.jsx

```diff
@@ -37,12 +38,8 @@ function SigningConfiguration({ t, signingConfiguration }) {
                                 {t('accountInfo.extendedPublicKey')}
                                 {signingConfiguration.xpubs.length > 1 && (' #' + (index+1))}
                             </strong><br />
-                            <QRCode data={xpub} /><br />
-                            <Input
-                                readOnly
-                                className={style.addressField}
-                                onFocus={focus}
-                                value={xpub} />
+                            <QRCode data={xpub} />
+                            <CopyableInput value={xpub} />
                         </div>
                     );
                 })
```

Comment:
Add a semicolon four lines below (where I cannot comment). This was the reason why CI failed.